assignment/scripts/array-practice.js

```diff
@@ -31,15 +32,14 @@ console.log('--- 3. Accessing items in an array ---');
 console.log('First animal is', animalArray[0])
 
 // 3.a. TODO: Log the second animal in the array 
-
+console.log('Second animal is', animals[1])
 
 // 3.b. TODO: Log the last animal in the array using it's array index 
-
+console.log('Last animal is', animals[3])
 
 // 3.c. (STRETCH) TODO: Log the last animal by using the array length, 
 //      instead of the exact index number of the last item
-
-
+console.log('Last animal is', animals[animals.length])
```

Comment:
Not quite. If you look at your console you will see that this logs `undefined`. This array has a `length` of 4, but there is no element at `animals[4]`.